iron-multi-selectable.html

```diff
@@ -108,6 +115,12 @@
         this.splice('selectedValues',i,1);
       }
       this._selection.setItemSelected(this._valueToItem(value), unselected);
+    },
+
+    _valuesToItems: function(values) {
+      return (values == null) ? null : values.map(function(value) {
```

Comment:
I tried to follow the pattern [here](https://github.com/PolymerElements/iron-selector/blob/master/iron-selectable.html#L256)